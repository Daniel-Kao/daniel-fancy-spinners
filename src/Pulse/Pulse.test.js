import React from 'react'
import renderer from 'react-test-renderer'
import { Pulse } from '..'

it('renders correctly', () => {
  const tree = renderer.create(<Pulse />).toJSON()
  expect(tree).toMatchSnapshot()
})
