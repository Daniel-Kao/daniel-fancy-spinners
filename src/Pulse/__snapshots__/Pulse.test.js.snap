// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="loadingio-spinner-pulse-en9z9u30elk"
>
  <div
    className="ldio-0brxcl0tnpnt"
  >
    <div />
    <div />
    <div />
  </div>
</div>
`;
